.github/workflows/ci.yml

```diff
@@ -21,3 +21,72 @@ jobs:
           aws-secret-access-key: ${{ secrets.DEV_SERVER_AWS_SECRET_ACCESS_KEY }}
       - run: aws sts get-caller-identity
       - run: npm run test:ci
+
+  # XXX TODO: It'd be nice to avoid the rebuild on Heroku and instead deploy
+  # the artifacts (source code + generated files + node_modules/) already built
+  # above.  This would dramatically reduce deploy times and move us closer to
+  # "deploy what you tested", but it may also run into platform compatibility
+  # issues given CI is building on a different platform (arch + OS + sys libs)
+  # than Heroku's dynos and some deps are compiled.  But should try it and see!
+  # Or do our build above inside a Heroku buildpack…
+  #   -trs, 2 May 2022
+  deploy:
+    if: |2
```

Comment:
What's the `2` for? I understand `|` is for the following multi-line expression.

Reply:
`2` explicitly sets the indent level of the multi-line block, which is necessary since the first line is indented slightly more than the remaining lines for easier scanning. You can play around with the various multi-line block options YAML provides at https://yaml-multiline.info.